common/maplib/map.mjs

```diff
@@ -255,6 +255,7 @@ class GameMap {
    * @return {String} cell - The raw content of the cell. '' if any error.
```

Comment:
I think we should return null on error, which is the standard for no value in our map. We'll need to update this comment as well.